read more separator (#21)
maelle/goodpress: man/rmdhunks/usage.Rmd

See [example post with math](https://rmd-wordpress.eu/post-slug/) and [its source](https://github.com/maelle/goodpress/tree/main/inst/post-example).

### Read more separator

You can add a [More tag](https://wordpress.com/support/wordpress-editor/blocks/more-block/2/) for your post if you don't want to show the whole post on your homepage.

> This can be done by adding a `HTML` code chunk in your `index.Rmd` post file:

Reviewer: ```suggestion
This can be done by adding a [Pandoc raw attribute](https://pandoc.org/MANUAL.html#extension-raw_attribute) in your `index.Rmd` post file i.e. copy-paste the lines below where you want the More tag to appear:
```

> 

Reviewer: ```suggestion

````markdown
```

```{=html}
<!-- wp:more -->
<!--more-->
<!-- /wp:more -->

> ```

Reviewer: ```suggestion
```
````
```
